ppr-ui/src/components/tables/mhr/SearchedResults.vue

```diff
@@ -406,13 +441,17 @@ export default defineComponent({
         return result.includeLienInfo !== true ? { ...result, includeLienInfo: false } : result
       })
 
+      // sort the results on the Review screen (all results will be sorted, not only unique)
+      if (props.isReviewMode) {
+        // sort based on the search type
+        localState.results = sortBy(localState.results, [UIMHRSearchTypeMap[localState.searchType]])
+      }
+
```

Comment:
I believe we will have to sort by mhrNumber to maintain the grouping, lets test once conflicts are resolved

Reply:
Yes, they asked for group sorting, but since it's one results list, I'm going to discuss this with Scott.